mobile/src/main/java/org/openhab/habdroid/core/OpenHabApplication.kt

```diff
@@ -59,6 +59,7 @@ class OpenHabApplication : MultiDexApplication() {
         ConnectionFactory.initialize(this)
         BackgroundTasksManager.initialize(this)
         RemoteLog.initialize(this)
+        BackgroundTasksManager.triggerPeriodicWork(this)
```

Comment:
This is the wrong place. Periodic tasks may trigger themselves over and over again.